aem/gradle/common.gradle.kts

```diff
@@ -11,8 +11,11 @@ allprojects {
             config {
                 // amend if needed
             }
+            docker {
+                stackName = "poc-stack"
+                composeFilePath = "local-environment/docker-compose.yml" // that's default value
```

Comment:
`gradle/local-environment/docker-compose.yml`

Reply:
Will be:

aem/gradle/instance
aem/gradle/environment